package.json

```diff
@@ -24,6 +24,7 @@
     "level": "8.0.1",
     "lodash": "4.17.21",
     "long": "5.2.3",
+    "queue-microtask": "^1.2.3",
```

Comment:
This is not a new dependency, we already have this as an indirect dependency and i'm adding as a direct deendency.